comp/api/api/utils/stream/stream.go

```diff
@@ -93,3 +96,23 @@ func GetStreamFunc(messageReceiverFunc func() MessageReceiver, streamType, agent
 		}
 	}
 }
+
+// OpenFileForWriting opens a file for writing
+func OpenFileForWriting(filePath string) (*os.File, *bufio.Writer, error) {
+	log.Infof("Opening file %s for writing logs. This file will be used to store streamlog output.", filePath)
```

Comment:
This log line seems too specific to logs streaming, you should probably log that before the call (the function is very generic)